testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -14,14 +14,24 @@ Feature: Be able to manage KVM virtual machines via the GUI
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  Scenario: Create KVM activation key
```

Comment:
Do we need this key for something special?
I would say we can reuse "Test-Channel-x86_64" right?

Reply:
There were issues with using the same activation key with the base product if I recall correctly and it needed its own key (or one of the others keys whicj would have to be renamed), I'll check again what the issue was though

Reply:
Keeping it that way for now since proxy product might be involved.